### DISCUSSION TITLE
How did the cool front page of https://squidfunk.github.io/mkdocs-material/ get generated?

### DISCUSSION THREAD

**Discussion by boozedog** [squidfunk/mkdocs-material#5465] [Q&A] [answered]
Does the front page of https://squidfunk.github.io/mkdocs-material/ get generated from mkdocs-material? If so, how?

**Comment by squidfunk** [comment, marked as answer]
It's a custom build ☺️ This has been asked multiple times in the past:

- #4687
- #4262

**Comment by boozedog** [comment]
Ahhh thanks @squidfunk I had missed those prior answers. I'll have a look at the insiders code. Thanks again :+1:
